wrappers/javascript/README.md

```diff
@@ -0,0 +1,38 @@
+# Indy VDR JavaScript Wrapper
+
+This is a JavaScript wrapper around the Indy VDR library. It is intended to be used in both Node.js and React Native environments.
+
+This document focuses on developer documentation for the wrapper. For usage see the specific readmes of the packages.
+
+## Setup
+
+### Prerequisites
+
+- Node.js 16.11.0 or higher
+- Yarn
+
+### Installation
+
+```sh
+yarn install
+```
+
+### Running Tests
+
+First make sure you've built the library using the `build.sh` script in the root of the repository.
+
+Then run the tests with the following command. Replace `<absolute-path>` with an absolute path to repository root.
```

Comment:
relatives also work IIRC which makes the command static :). not exactly sure what the relative path is from where we want to run the tests. (maybe we can do `yarn run:with-build` that will add the env var to `yarn test`).

Reply:
Yes, had some issues with this as the base path is kinda weird with yarn workspaces, but the custom command is smart!